# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1548356800 0
#      Thu Jan 24 19:06:40 2019 +0000
# Branch stable
# Node ID d3600d0511d8bde6f75ef3a1cf5d2332cda3dc24
# Parent  3945d8dd56a1ea60d998148684f7ae3c29880858
* .hgignore: Also ignore build-aux/ar-lib.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -30,16 +30,17 @@ syntax: regexp
 
 ## Emacs associated files
 (^|/)TAGS$
 (^|/)semantic.cache$
 
 ## Scripts associated with building Octave
 ^build-aux/config\.(guess|rpath|sub)$
 (^|/)build-aux/check-subst-vars\.sh$
+^build-aux/ar-lib$
 ^build-aux/depcomp$
 ^build-aux/install-sh$
 ^build-aux/ltmain\.sh$
 ^build-aux/mdate-sh$
 ^build-aux/missing$
 ^build-aux/move-if-change$
 (^|/)build-aux/subst-config-vals\.sh$
 (^|/)build-aux/subst-cross-config-vals\.sh$
